kuma/wiki/tests/test_views_code.py

```diff
@@ -105,40 +55,3 @@ def test_code_sample_host_restricted_host(code_sample_doc, settings, client):
     assert response.status_code == 200
     assert "public" in response["Cache-Control"]
     assert "max-age=31536000" in response["Cache-Control"]
-
-
-def test_raw_code_sample_file(
```

Comment:
I think we want to keep this test, since the endpoint will remain at least a while longer.